libnopegl/src/node_draw.c

```diff
@@ -166,26 +168,80 @@ static const struct node_param render_params[] = {
     {NULL}
 };
 
-static int render_init(struct ngl_node *node)
+static int check_params(struct ngl_node *node)
```

Comment:
I think the node can be `const` if you're only checking parameters?

Reply:
Done !